apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -29,10 +32,41 @@ const useSections = () => {
   return [sections, updateSection];
 };
 
+const saveSection = (e, section) => {
+  e.preventDefault();
+
+  const form = document.querySelector(`#${FORM_ID}`);
+  if (!form.checkValidity()) {
+    form.reportValidity();
+    return;
+  }
+
+  const csrfToken = document.querySelector('meta[name="csrf-token"]')
```

Comment:
i saw we were using jquery for this elsewhere, but it was easier to stub `document.querySelector`, so I just used the builtins here. is that ok?

Reply:
as long as it works, seems fine to me